software/spmd/common/asm.ld

```diff
@@ -18,6 +18,7 @@ SECTIONS
   _bsg_dram_start_addr = 0x80000000;
   .text 0x80000000 :  {
           *(.text)
+          *(.dram)
```

Comment:
what does adding this line do? to me it looks like it places .dram after .text, but wasn't that happening anyway?

Reply:
To reproduce the bug in smallest possible test case, I chose to write an assembly program. But the assembly linker, spmd/common/asm.ld, didn't include the section for .dram section. Hence I added this to mimic that of current C linker.

Reply:
Great!